changelog_unreleased/blog-post-intro.md

```diff
@@ -1,7 +1,7 @@
 ---
-author: "🚧"
-authorURL: "https://github.com/🚧"
-title: "Prettier 🚧"
+author: "Sosuke Suzuki (@sosukesuzuki)"
+authorURL: "https://github.com/sosukesuzuki"
+title: "Prettier 2.2: new JavaScript parsers, TS 4.1 and ESM standalone bundles"
 ---
 
-🚧 Write an introduction here.
+This release supports new JavaScript parsers [espree](https://github.com/eslint/espree) and [meriyah](https://github.com/meriyah/meriyah), supports [TypeScript 4.1](https://devblogs.microsoft.com/typescript/announcing-typescript-4-1-rc/), ships ESM standalone bundles for modern browsers, and includes many bug fixes and improvements!
```

Comment:
Shouldn't this file unchanged?

Reply:
This file is used to generate a draft (https://github.com/prettier/prettier/blob/master/scripts/draft-blog-post.js#L81). Also this was used for 2.0 blog (https://github.com/prettier/prettier/pull/7417/files#diff-15f290566c72ae2b4a5ef5b40d9850ddbeeb55e260db4e0a653332d8f603cb6a).

Reply:
I know that, but isn't this file a template? Or will it restore at release?

Reply:
I'll fix this file when I clear changelog_unreleased, just as when 2.0 is released (https://github.com/prettier/prettier/commit/8f7164fdf49c8241ffaca2f1c7cc4a3f6af06e2a#diff-15f290566c72ae2b4a5ef5b40d9850ddbeeb55e260db4e0a653332d8f603cb6a).